tests/unit/test_main.py

```diff
@@ -68,20 +68,10 @@ def test_basic_inbound(self):
         # we exited with the return value of bisect_inbound
         self.assertEquals(exitcode, 0)
 
-    def test_inbound_revs_must_be_given(self):
```

Comment:
Instead of removing this test, you can just change the arglist variable by removing all the --inbound in it, eg:
argslist = [['--good-rev=1'], ['--bad-rev=5']]
The test must then pass again (given that you changed the "and" to an "or" like I said in my last comment)